front50-sql/src/main/resources/db/changelog/20191030-initial-plugins-schema.yml

```diff
@@ -0,0 +1,47 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-plugins-table
+      author: jonsie
+      changes:
+        - createTable:
+            tableName: plugins
+            columns:
+              - column:
+                  name: id
+                  type: char(255)
```

Comment:
The `id` property on plugins is sha256. This should be `char(64)`.

Reply:
Good catch.